inc/class_parser.php

```diff
@@ -1220,12 +1220,23 @@ function mycode_parse_url($url, $name="")
 	*/
 	function mycode_parse_font_callback($matches)
 	{
-		// Replace any occurrence(s) of double quotes in fonts with single quotes.
-		// A back-fix for double-quote-containing MyBB font tags in existing
-		// posts prior to the client-side aspect of this fix for the
-		// browser-independent SCEditor bug of issue #4182.
-		$fonts = str_replace('"', "'", $matches[2]);
-
+		$fonts = explode(',', $matches[2]);
```

Comment:
This seems to assume that commas do not (cannot) occur within quoted font families. Given my reading of that same MDN article, this assumption is false.